src/pivot-table/components/PivotTable.tsx

```diff
@@ -68,6 +72,13 @@ export const StickyPivotTable = ({
     }
   }, [layoutService]);
 
+  useLayoutEffect(() => {
+    if (scrollableContainerRef.current && !dataModel.isLoading) {
+      scrollableContainerRef.current.scrollLeft = 0;
+      scrollableContainerRef.current.scrollTop = 0;
+    }
+  }, [pageInfo.currentPage, dataModel.isLoading]);
```

Comment:
Why `dataModel.isLoading` here?

Reply:
previously we needed to wait for that then change the page, probably not needed any more, since the fetching is not happening with dataModel.